python_graphql_client/graphql_client.py

```diff
@@ -37,6 +37,7 @@ def execute(
         variables: dict = None,
         operation_name: str = None,
         headers: dict = {},
+        riase_for_status: bool = True,
```

Comment:
By chance did you mean?

```suggestion
        raise_for_status: bool = True,
```